src/update.c

```diff
@@ -190,7 +190,7 @@ static int update_loop(struct list *updates, struct manifest *server_manifest)
 	return ret;
 }
 
-int add_included_manifests(struct manifest *mom, struct list **subs)
```

Comment:
The patch set isn't meant to be applied partially...I can change it if needed, but for reviewer's sake I broke it up a lot more so it was easier to look through.